w2w/w2w.py

```diff
@@ -1528,17 +1552,30 @@ def checks_and_cleaning(info: Info, ucp_table: pd.DataFrame, nbui_max: float) ->
     dst_data_orig = xr.open_dataset(info.dst_file)
     orig_num_land_cat = dst_data_orig.NUM_LAND_CAT
     urban_cat = dst_data_orig.ISURBAN
-    if urban_cat in da.LU_INDEX.values:
+
+    #Define pixels that are urban in the original data
+    if orig_num_land_cat == 61:
+        LCZ_URBAN = [51, 52, 53, 54, 55, 56, 57, 58, 59, 60]
+        urban_cat_list = LCZ_URBAN + [urban_cat]
+    elif orig_num_land_cat == 41:
+        LCZ_URBAN = [31, 32, 33, 34, 35, 36, 37, 38, 39, 40]
+        urban_cat_list = LCZ_URBAN + [urban_cat]
+    else:
+        urban_cat_list = [urban_cat]
+
```

Comment:
same comment here as above ...

Reply:
Yes we can clean that a little bit. Perhaps using Info.LCZ_BUILT and dst_data_orig.NUM_LAND_CAT. But not sure how to deal with those options you mention: LCZ 11 and LCZ 15